packages/web/src/tools/_forms.scss

```diff
@@ -28,7 +28,7 @@
         @include typography.generate(map.get($theme, input-typography));
 
         display: block;
-        max-width: 100%;
+        width: 100%;
```

Comment:
This would override the `size` attribute. Is it OK?

Reply:
As discussed in person:

- `TextField` will have a default width of 288 px / 18 rem so it's in sync with Figma.
- `TextField` box model will change to `inline-block` so multiple fields can be placed in a row.
-  The `<input>` itself should be always 100% wide, it's width will be controlled by its containing `<div>`.
- There will be a new API option `isFluid` (?) for both `TextField` and `Button` so their width can be controlled by their container.
- We will postpone the support of the `size` attribute for later as it becomes a bit more difficult with the changes proposed.

Reply:
There is a `Button-block` class to set `width: 100%` on a button. So I think that it is what we need, though.